autoapi/mappers/dotnet.py

```diff
@@ -198,9 +212,9 @@ def __init__(self, obj, **kwargs):
         if syntax is not None:
             # Code example
             try:
-                self.example = syntax['content']['CSharp']
-            except KeyError:
-                pass
+                self.example = syntax['content']
+            except (KeyError, TypeError):
+                traceback.print_exc()
```

Comment:
Not sure if there's a better pattern here, but I like at least seeing the errors, instead of having them pass silently. Perhaps we should be using the sphinx error logger here, or something.